src/devices/Ad7193/Ad7193.cs

```diff
@@ -0,0 +1,842 @@
+            }
+
+            set
+            {
+                SetRegisterBits(Register.Mode, BitMask.ModeDAT_STA, value ? 1U : 0);
+
+                if (value)
+                {
+                    // change register size to 4, because the 8-bit Status register is now appended to the 24-bit ADC value
+                    _registerSize[(byte)Register.Data] = 4;
+                }
+                else
+                {
+                    // change register size to 3, because we have a 24-bit ADC value (without the 8-bit Status register)
+                    _registerSize[(byte)Register.Data] = 3;
+                }
+            }
+        }
+
+        /// <summary>
+        /// True if the jitter correction is enabled in the driver. Jitter correction tries to guarantie the steady samples rate even is the hardware rasterization speed jitters.
+        /// </summary>
+        public bool JitterCorrection { get; set; }
+
+        /// <summary>
+        /// Switches from differential input to pseudo differential inputs.
+        /// When the pseudo bit is set to 1, the AD7193 is configured to have eight pseudo differential analog inputs. When pseudo bit is set to 0, the AD7193 is configured to have four differential analog inputs.
+        /// </summary>
+        public AnalogInputMode InputMode
+        {
+            get
+            {
+                return (AnalogInputMode)(GetRegisterBits(Register.Configuration, BitMask.ConfigurationPseudo));
+            }
+
+            set
+            {
+                SetRegisterBits(Register.Configuration, BitMask.ConfigurationPseudo, (uint)value);
+            }
+        }
+
+        /// <summary>
+        /// Changes the currently selected channel
+        /// </summary>
+        public Channel ActiveChannels
+        {
+            get
+            {
+                return (Channel)(GetRegisterBits(Register.Configuration, BitMask.ConfigurationCH));
+            }
+
+            private set
+            {
+                SetRegisterBits(Register.Configuration, BitMask.ConfigurationCH, (uint)value);
+            }
+        }
+
+        /// <summary>
+        /// Sets the amount of averaging. The data from the sinc filter is averaged by 2, 8, or 16. The averaging reduces the output data rate for a given FS word; however, the RMS noise improves.
+        /// </summary>
+        public AveragingMode Averaging
+        {
+            get
+            {
+                return (AveragingMode)(GetRegisterBits(Register.Mode, BitMask.ModeAVG));
+            }
+
+            set
+            {
+                SetRegisterBits(Register.Mode, BitMask.ModeAVG, (uint)value);
+            }
+        }
+
+        /// <summary>
+        /// Set the filter output data rate select bits. The 10 bits of data programmed into these bits determine the filter cutoff frequency, the position of the first notch of the filter, and the output data rate for the part.
+        /// </summary>
+        public ushort Filter
```

Comment:
Please rename it to Low/HighPassFilterHz and make units clear. If unit is not Hz or multiple it would be better if you did translation here